internal/inventory/inventory.go

```diff
@@ -0,0 +1,13 @@
+package inventory
+
+import (
+	"kg/procurement/internal/vendors"
+)
+
+type Item struct {
```

Comment:
Are you sure that object item only has this information?

Reply:
Thank you for your question. For now, the `Item` object is still provisional as we need additional information from Kompas Gramedia